.github/workflows/xrt_master.yml

```diff
@@ -591,5 +591,16 @@ jobs:
           github-server-url: ${{ secrets.SERVER_URL }}
           sshKey: ${{ secrets.CI_PRIVATE_SSH_KEY }}
           runnerName: ${{ runner.name }}
-  
 
+  notify-slack:  
+    needs: [build, windows-build, apu-package-build, package-download, extract-platforms, pcie-hw-tests, pcie-sw-emulation-tests, pcie-hw-emulation-tests, test-summary, setenv-download, Release_to_xco, Release_to_all_sites, Upload_packages_to_Artifactory, gradle-artifactory-upload]  
```

Comment:
Do we need to add all jobs name? can't we just use needs "gradle-artifactory-upload" ?

Reply:
Let me test that

Reply:
actually we would need "Release_to_all_sites" and "gradle-artifactory-upload"

Reply:
Yes, now I added dependency on "Release_to_all_sites" and "gradle-artifactory-upload"